ibmcloud/README.md

```diff
@@ -444,4 +454,25 @@ groups, subnet and gateway with:
 ```bash
 $ cd ibmcloud/terraform/common
 $ terraform destroy
-```
\ No newline at end of file
+```
+
+## Troubleshooting
+
+### Cluster creation when switching use `ssh_pub_key`
+
+If during the process of setting up the cluster you may experience the following error regarding your SSH key
+> Error: Error deleting SSH Key : SSH-key still in use
+
+Which happens when you have provided your SSH public key via the `ssh_pub_key` Terraform variable and successfully running `terraform apply` but running `terraform apply` subsequently results in the above error.
+
+This is caused by the IBM Cloud terraform provider forcing the replacement of the SSH key resource. As such Terraform may try to delete the existing SSH key while the Virtual Server Instances are using it.
+
+To resolve this issue:
+
+- Use `terraform state rm ibm_is_ssh_key.created_ssh_key[0]` to remove the SSH key resource from Terraform's internal state.
+- Delete the ssh_pub_key variable from the terraform.tfvars file
```

Comment:
To be consistent with elsewhere we would want to put ssh_pub_key and terraform.tfvars in backticks to format them as code in Markdown.